src/store/modules/shots.js

```diff
@@ -141,6 +142,9 @@ const cache = {
 }
 
 const helpers = {
+  getCurrentProduction () {
```

Comment:
This is used solely in the `helpers` section on the module. So the only other solution is to pass the `currentProduction` in each method of the `helpers` object.

I've also done that because this method exists in `store/modules/asset.js`

What would you prefer?

Reply:
Ok got it. It was not clear when I read the diff. Thank you.